client/src/components/UserProfile.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const UserProfile = ({ user }) => {
+    return (
+        <div>
+            <h1>This is a page built for a user to see</h1>
```

Comment:
we should avoid pages or implementations that are "placeholders". It's fine if more info will be added to this page overtime, but we need to think of the current feature this provides to users. Right now this does not show them any information. At the very least, this should start to display information about the user on their profile - like their email, etc.